app/server/me_api/credit_transfer.py

```diff
@@ -292,6 +297,21 @@ def post(self):
                                              transfer_use=transfer_use,
                                              transfer_mode=transfer_mode,
                                              uuid=uuid)
+            # Check the insentives to make sure that all of them pass, but don't actually make the payments yet (do_not_transact). 
+            # We want to make sure that all the transfers are valid before we allow them!
+            for incentive in g.active_organisation.incentives: # Do we want to use the active org, or the recipients org?
```

Comment:
Making a note on this question

Reply:
goooood question. i mean they're generally one and the same thing, but probably active org?